raiden-network/light-client
# Transaction history

**Thank you for submitting this pull request :)**

Fixes #1212  
Fixes #1451 

**Short description**
Adds a list with a user's transaction history. The transactions are lazy-loaded.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Make a transfer and the latest transfer should appear on top of the transaction history list.
2. Check layout (Mobile, different resolutions)
